events/event_handler.go

```diff
@@ -160,6 +161,18 @@ func (fh *eventHandler) handleStatusEvent(body []byte) error {
 		"TaskStatus": task.TaskStatus,
 	}).Info("Got StatusEvent")
 
+	app, err := fh.marathon.App(task.AppID)
```

Comment:
problem is that we have task managed by tool _mesos-consul_ when tag `consul` isn't present. and tasks with tag `consul` managed by _marathon-consul_ (as a transition step).

We really don't want to see _marathon-consul_ deregistering apps managed by _mesos-consul_.